compose/production/django/Dockerfile

```diff
@@ -15,12 +15,12 @@ RUN apt-get update \
   && rm -rf /var/lib/apt/lists/*
 
 RUN addgroup --system django \
-    && adduser --system --ingroup django django
+  && adduser --system --ingroup django django
```

Comment:
is this a real diff? If not can we get rid of it?

Reply:
its vscode autoformatting, ill just undo that